e2e/setup.sh

```diff
@@ -3,5 +3,5 @@
 . lib.sh
 
 setup_suite() {
-	create_cluster
+	create_cluster "$(build_kind_config 2)"
```

Comment:
This was the change we needed to make the tests pass after rebasing on the cluster-reutilization PR. Here we create a cluster with 2 workers. Before, we were defaulting to the basic config without any workers, so the adjacency matrix did not match what we expected